pkg/pxc/app/binlogcollector/binlog_collector_test.go

```diff
@@ -0,0 +1,170 @@
+package binlogcollector
+
+import (
+	"reflect"
+	"testing"
+
+	api "github.com/percona/percona-xtradb-cluster-operator/pkg/apis/pxc/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestGetDeploymentLabelsAndMatchLabels(t *testing.T) {
```

Comment:
The function name we are testing is `GetDeployment`, so our test should be named `TestGetDeployment`. The other details are not needed since we have individual tests cases for labels and we could also test other things on the deployment construct as well.